drools/src/main/java/com/datatorrent/drools/rules/DroolsRulesReader.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2017 DataTorrent, Inc. ALL Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.drools.rules;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.drools.core.io.impl.InputStreamResource;
+import org.kie.api.KieServices;
+import org.kie.api.builder.KieBuilder;
+import org.kie.api.builder.KieFileSystem;
+import org.kie.api.builder.Message;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+public class DroolsRulesReader implements AbstractRulesReader
+{
+  private static Logger LOG = LoggerFactory.getLogger(DroolsRulesReader.class);
+
+  @Override
+  public void loadRulesFromDirectory(String rulesDir) throws IOException
+  {
+    KieFileSystem kieFileSystem = getKieFileSystem();
+    FileSystem sourceFileSystem = getFSInstance(rulesDir);
```

Comment:
I m removing this method and instead adding the lines to get KieFileSystem instance to "loadRulesInKieFileSystem" method.